FWCore/Framework/src/global/EDAnalyzerBase.cc

```diff
@@ -179,6 +180,7 @@ namespace edm {
     }
     
     void EDAnalyzerBase::preallocStreams(unsigned int) {}
+    void EDAnalyzerBase::preallocate(unsigned int) {}
```

Comment:
Let's change these to take the full `PreallocationConfiguration` object. Otherwise, how would someone know what the `unsigned int` represents.

Reply:
Done, I changed the argument to preallocate and preallocStreams.